diagnostic_tool/sim2real_diff_plotter.py

```diff
@@ -0,0 +1,41 @@
+import dill
```

Comment:
Curious why `dill` instead of `pickle`?

Reply:
`dill` is a superset of `pickle`, so is more robust.